omod/src/main/java/org/openmrs/module/attachments/page/controller/AttachmentsPageController.java

```diff
@@ -1,9 +1,5 @@
 package org.openmrs.module.attachments.page.controller;
```

Comment:
Hi @mogoodrich , I think you missed an earlier comment from @mks-d . This controller is no longer used and it should be removed from the code base.

Reply:
@mks-d  @cioan  isn't in the "AttachmentsController" that is no longer used?

(Though I did modify that as well, see below, because I changed a method signature so needed to in order to compile.  Fine with deleting it entirely if we want)